pandas/core/computation/pytables.py

```diff
@@ -132,6 +132,8 @@ def pr(left, right):
             res = pr(left.value, right.prune(klass))
         elif not (is_term(left) or is_term(right)):
             res = pr(left.prune(klass), right.prune(klass))
+        else:
+            res = None
```

Comment:
shouldnt the previous 4 cases cover all possibilities?

Reply:
I think it's cleaner to cover all possibilities, so res can never be undefined. Or I can put `res = None` above.

Reply:
then change the elif on 133 to `else:` with a comment "# i.e. `not (is_term(left) or is_term(right))`"

Reply:
That makes more sense indeed, will change.